### PR TITLE
Making tanh squashing the default

### PR DESCRIPTION
**Pull request by vincentpierre**
### Proposed change(s)

This PR is to use tanh transformation by default with PPO (already the default with SAC)
To improve stability and prevent the apparition of NaNs the following changes were made.

 - Clamp the log_probs of the TanhGaussian between -20 and 20. (Since in PPO, the log probabilities are exponentiated, we cannot allow log probs to take too small or too large values)
 - Initialize the log_sigma variable to -1.1. Since we do not scale the normal distribution by dividing it by 3 anymore, we need to reduce the standard deviation by a factor of 3 which is the same as lowering log_sigma to -1.1 (1.1 = log(1/3))

I am still running a retraining job with these changes, but initial results are encouraging. 

### Types of change(s)

- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: ml-agents/mlagents/trainers/torch/distributions.py [hunk 1]
```diff
@@ -175,9 +179,10 @@ def forward(self, inputs: torch.Tensor) -> List[DistInstance]:
             # Expand so that entropy matches batch size. Note that we're using
             # mu*0 here to get the batch size implicitly since Barracuda 1.2.1
             # throws error on runtime broadcasting due to unknown reason. We
-            # use this to replace torch.expand() becuase it is not supported in
+            # use this to replace torch.expand() because it is not supported in
             # the verified version of Barracuda (1.0.X).
-            log_sigma = mu * 0 + self.log_sigma
+            # log_sigma = mu * 0 + self.log_sigma
```
Comment (awjuliani) [inline ml-agents/mlagents/trainers/torch/distributions.py:184]: Should this commented line be removed?

**Review by awjuliani — COMMENTED** [review]
Code looks good. So long as we can show that it does not result in a performance regression, I approve.

File: ml-agents/mlagents/trainers/torch/distributions.py (hunk 1 above)
Comment (vincentpierre) [reply ml-agents/mlagents/trainers/torch/distributions.py:184]:
```suggestion
```

**Comment by vincentpierre** [comment]
Some comparison of results in [this doc](https://docs.google.com/spreadsheets/d/1dO1k3FZQyZo-lhx0iETC_ZMhn9Skhe0e3fgOSYplB8U/edit#gid=0)

**Pull request closed without merging**